source/routing/raptor.cpp

```diff
@@ -224,15 +248,81 @@ make_starting_points_snd_phase(const RAPTOR& raptor,
                 count,
                 (clockwise ? working_labels.dt_pt(a.first) + walking_t
                            : working_labels.dt_pt(a.first) - walking_t),
-                walking_t
+                walking_t,
+                false
             };
-            res.add(starting_point);
+            overfilter.add({res.size(), starting_point});
+            res.push_back(starting_point);
         }
     }
 
+    for (const auto& pair : overfilter) {
+        res[pair.first].has_priority = true;
+    }
+
+    std::sort(res.begin(), res.end(), CompSndPhase(clockwise)); // most interesting solutions first
+
     return res;
 }
+
+Journey convert_to_bound(const StartingPointSndPhase& sp,
+                         uint32_t lower_bound_fb,
+                         uint32_t lower_bound_conn,
+                         bool clockwise) {
+    Journey journey;
+    journey.sections.resize(sp.count);
```

Comment:
The dominance function call size to count the number of sections in the journey.